Description: Bump bindgen to v0.71 or higher
 Bindgen v0.71 only supports rustc v1.33 and higher
Author: NoisyCoil <noisycoil@debian.org>
Forwarded: not-needed
Last-Update: 2024-12-29
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -46,7 +46,7 @@
 default-features = false
 
 [build-dependencies.bindgen]
-version = "0.69.0"
+version = ">=0.71, <1.0"
 optional = true
 
 [build-dependencies.cc]
--- a/build.rs
+++ b/build.rs
@@ -243,7 +243,7 @@
 ) {
     #[allow(deprecated)]
     let mut builder = bindgen::Builder::default()
-        .rust_target(bindgen::RustTarget::Stable_1_28)
+        .rust_target(bindgen::RustTarget::Stable_1_33)
         .size_t_is_usize(true)
         .use_core()
         .ctypes_prefix("libc")
